### PR TITLE
EzInstall

### PR DESCRIPTION
**Pull request by armintoepfer**
Add an easy way to compile pbccs without dependency juggling.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,7 +52,7 @@ if(NOT HTSLIB_INCLUDE_DIRS OR
                      REQUIRED)
     else()
         find_package(htslib
-                     PATHS ${PacBioCCS_SOURCE_DIR}/../../../lib/cpp/htslib
```
Comment (evolvedmicrobe) [inline CMakeLists.txt]: Am I reading this correctly that we had one variable which was PacBioCCS_SOURCE_DIR and another that was PacBioCCS_SourceDir? (or are variable names case/underscore insensitive?)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,19 @@
 
 The ccs program takes multiple reads of the same SMRTbell sequence and combines them to produce one high quality consensus sequence.
 
+## Quick Install
+
+Boost, cmake, and c++ compiler are required for the quick installation process:
+
+```
+git clone https://github.com/PacificBiosciences/pbccs && cd pbccs
+git pull --rebase origin master && git submodule update --init
```
Comment (evolvedmicrobe) [inline README.md]: Given that they have just done a git clone, why the need for the `pull --rebase origin master`?

File: CMakeLists.txt (hunk 1 above)
Comment (armintoepfer) [inline CMakeLists.txt]: They are different. SOURCE_DIR is the root and SourceDir is src/

Comment (evolvedmicrobe) [inline CMakeLists.txt]: Okay, I know it's unrelated to this SourceDir variable is our own creation and not required by CMAKE right? I generally don't like having such similar variable names floating around so might create a separate pull to rename that.

File: README.md (hunk 2 above)
Comment (armintoepfer) [inline README.md]: Good question, maybe that's more a line for updating the source and not necessary here.

**Pull request merged**